Language/Functions/Communication/Serial/serialEvent.adoc

```diff
@@ -14,7 +14,7 @@ title: serialEvent()
 
 [float]
 === Description
-Called when data is available. Use `Serial.read()` to capture this data.
+Called at the end of loop() when data is available. Use `Serial.read()` to capture this data.
```

Comment:
```suggestion
Called at the end of link:../../../../structure/sketch/loop[`loop()`] when data is available. Use `Serial.read()` to capture this data.
```

- Use [standard formatting](https://github.com/arduino/reference-en/blob/master/AsciiDoc_sample/AsciiDoc_Dictionary/AsciiDoc_Template-Dictionary.adoc) for inline code
- Link to [`loop()` reference page](https://www.arduino.cc/reference/en/language/structure/sketch/loop/)

Reply:
Hi @mtedaldi. I wanted to check back in with you. If you will update the pull request according to the instructions I provided above I will be happy to merge this valuable enhancement.